### PR TITLE
fix: correctly setup husky

### PR DESCRIPTION
**Pull request by sakulstra**
seems like at some point husky was updated, but we never updated the config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -115,15 +115,13 @@
     "start:dev": "react-app-rewired start",
     "build": "GENERATE_SOURCEMAP=false react-app-rewired build",
     "test": "react-app-rewired tests",
+    "lint:fix": "prettier --write src",
```
Comment (foodaka) [inline package.json:118]: this is useful

File: src/translations/fr.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
   "components.ConnectWalletModal.components.LedgerChecklist.more": "",
   "components.ConnectWalletModal.components.LedgerChecklist.secondCheck": "",
   "components.ConnectWalletModal.components.LedgerChecklist.thirdCheck": "",
+  "components.ConnectWalletModal.components.SelectPreferredNetwork.forkNetwork": "",
```
Comment (foodaka) [inline src/translations/fr.json:21]: @sakulstra do we manaully add all these to every single language file?

**Review by foodaka — APPROVED** [review]
There are other fixes besides husky in the PR. Is that what you want?

File: src/translations/fr.json (hunk 2 above)
Comment (sakulstra) [reply src/translations/fr.json:21]: no it's generated via this extract utility https://github.com/aave/aave-ui/pull/225/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R121 

This was usually executed via husky on precommit, but was broken for some time without someone noticing. This is why now when it's fixed these changes are in the pr.

**Comment by sakulstra** [comment]
> There are other fixes besides husky in the PR. Is that what you want?

Yes they are corelated, husky was broken which is why pre-commit was not executed.
I18n extract was broken due to babel config - not sure how this ever worked before.

With these two things fixed now it generated some changes which are intended as they were missing before.

**Pull request merged**